src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/get_accounts.py

```diff
@@ -90,6 +97,45 @@ def get_accounts():
     )
 
 
+def get_accounts_from_tags():
+    accounts = get_accounts()
+    tag_filters = []
+    for tag in TARGET_TAGS.split(','):
+        tag_name = tag.split('=')[0]
+        tag_value = tag.split('=')[1]
+        tag_filters.append({
+            "Key": tag_name,
+            "Value": tag_value})
+
+    LOGGER.info(
+        "Tag filters %s",
+        tag_filters
+    )
+
+    organizations = get_boto3_client(
+        'organizations',
+        (
+            f'arn:{PARTITION}:sts::{MANAGEMENT_ACCOUNT_ID}:role/'
+            f'{CROSS_ACCOUNT_ACCESS_ROLE}-readonly'
+        ),
+        'getaccountIDs',
+    )
+    filtered_accounts = []
+    for account in accounts:
+        tags = account['Tags'] = organizations.list_tags_for_resource(
+            ResourceId=account['AccountId']
+        )['Tags']
+        for tag_filter in tag_filters:
+            found = list(filter(lambda item: (
+                item["Key"] == tag_filter["Key"] and item["Value"] == tag_filter["Value"]), tags))
+            if len(found) > 0:
+                print(
```

Comment:
Please remove this `print` or replace by a `LOGGER` statement.

Reply:
Resolved with commit 646a3b485afe3fb56af23d20a8ccecca259b846b